packages/ffe-cards-react/src/GroupCard/GroupCard.tsx

```diff
@@ -0,0 +1,48 @@
+import React, { ForwardedRef } from 'react';
+import classNames from 'classnames';
+import { fixedForwardRef } from '../fixedForwardRef';
+
+type BgColor = 'sand-30' | 'sand-70' | 'frost-30' | 'syrin-30' | 'syrin-70';
+type BgColorDarkmode = 'natt' | 'svart' | 'koksgraa';
+
+export interface GroupCardProps
+    extends Omit<React.ComponentPropsWithoutRef<'div'>, 'children'> {
+    shadow?: boolean;
+    /** The children of the GroupCard component */
+    children: React.ReactNode;
```

Comment:
Sleng gjerne in en kommentar på dissse også. Det brukes vell til generert documentatiojn i det verktøy vi går før?